### PR TITLE
Add tests for outfits, fixes families outfits, Syndicate Ship Captain, and armored bounty hunters

### LINKED ISSUE #52105: Family cops don't spawn with guns.
Step 1. Play families
Step 2. Get wanted level to 2 or 3 stars
Step 3. Play cops
Step 4. Notice you have no gun. 

Presumably this is caused by the outfit having the shotgun on the belt. Which is impossible. So you don't spawn with one.

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
Supercedes #52209, along with making sure it never happens again. Also fixed the Syndicate Ship Captain, who previously did not receive a belt or an ID since they had no jumpsuit.

Fixes #52105. Closes #52209.

## Why It's Good For The Game
Coverage and guns are good.

## Changelog
:cl:
fix: Fixed SWAT and Armored beat cops spawning without a weapon.
fix: Fixed the Syndicate Ship Captain outfit not having a jumpsuit, ID, or belt.
fix: Armored fugitive bounty hunters now correctly start with an extra tank of plasma in their hand.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
You're making us look bad

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -713,6 +713,7 @@
 
 /datum/outfit/syndicatespace/syndicaptain
 	name = "Syndicate Ship Captain"
+	uniform = /obj/item/clothing/under/syndicate/tacticool
```
Comment (Fikou) [inline code/game/objects/structures/ghost_role_spawners.dm:716]: why the fake tacticool one and not the combat/regular one the other guys get

**Comment by optimumtact** [comment]
yes keep raising the standard so I can apply it to everyone else

File: code/game/objects/structures/ghost_role_spawners.dm (hunk 1 above)
Comment (Jared-Fogle) [reply code/game/objects/structures/ghost_role_spawners.dm:716]: I just figured that it should be different to make them stand out. I'll make it the combat one though on second thought.

**Comment by Jared-Fogle** [comment]
Someday, we'll get this up to -2%.

image

**Review by ShizCalev — COMMENTED** [review]
missed r_pocket & l_pocket

**Comment by Jared-Fogle** [comment]
image
😛

**Comment by Jared-Fogle** [comment]
> PASS: /datum/unit_test/outfit_sanity 11.8s

Pretty slow, though making it faster with just this test would likely require it to be less accurate, which I don't think is a good trade off. I wonder if initializing some outfits is just very slow.

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**